net/isc-dhcp/files/dhcpd.init

```diff
@@ -263,6 +265,11 @@ general_config() {
 	echo "max-lease-time $max_lease_time;"
 
 	[ -n "$domain" ] && echo "option domain-name \"$domain\";"
+
+	rm -f /tmp/resolv.conf
+	echo "# This file is generated by the DHCPD service" > /tmp/resolv.conf
+	echo "domain $domain" >> /tmp/resolv.conf
```

Comment:
Don't you have to check the domain name is non empty before echoing it to resolv.conf ?

Reply:
Fixed.